src/modules/twap/utils/getTwapOrdersItems.ts

```diff
@@ -0,0 +1,40 @@
+import { Order } from 'legacy/state/orders/actions'
+
+import { getTwapOrderStatus } from './getTwapOrderStatus'
+import { parseTwapOrderStruct } from './parseTwapOrderStruct'
+
+import { TwapToDiscreteOrders } from '../hooks/useDiscreteOrdersFromOrderBook'
+import { TwapOrdersSafeData } from '../services/fetchTwapOrdersFromSafe'
+import { TwapOrderItem, TwapOrderInfo, TwapOrdersAuthResult } from '../types'
+
+export function getTwapOrdersItems(
+  safeAddress: string,
+  ordersInfo: TwapOrderInfo[],
+  ordersAuthResult: TwapOrdersAuthResult,
+  discreteOrders: TwapToDiscreteOrders
+): TwapOrderItem[] {
+  return ordersInfo.map(({ safeData, id }) => {
+    return getTwapOrderItem(safeAddress, safeData, id, ordersAuthResult[id], discreteOrders[id])
+  })
+}
+
+function getTwapOrderItem(
```

Comment:
this method is called `get` but does it actually get anything?

Reply:
Renamed to `buildTwapOrdersItems`, thanks!